MDLIB/mdoodz-private.h

```diff
@@ -561,4 +561,12 @@ void            SetBCs(SetBCs_ff setBCs, MdoodzInput *instance, grid *mesh);
 
 void            ValidateSetup(MdoodzSetup *setup, MdoodzInput *instance);
 
+typedef struct {
+  double VxWestSum;
+  double VxEastSum;
+  double VzSouthSum;
```

Comment:
what about VzNorthSum? - to keep MDOODZ universally applicable? :)

Reply:
I don't need VzNorthSum here. It's a struct that we will use to pass in several parameters in one function argument